corehq/couchapps/attachments/views/attachments/map.js

```diff
@@ -10,6 +10,13 @@ function (doc) {
                     attachments[key] = doc._attachments[key];
                 }
             }
+            for (var key in doc.external_blobs) {
+                if (doc.external_blobs.hasOwnProperty(key) &&
+                    doc.external_blobs[key].content_type !== "text/xml") {
+                    media += 1;
+                    attachments[key] = doc.external_blobs[key];
+                }
+            }
```

Comment:
Because of other issues that have come up on this PR, I'm holding off on starting the preindex until I have resolved them.